src/share/poudriere/jail.sh

```diff
@@ -436,6 +445,72 @@ build_and_install_world() {
 	fi
 }
 
+install_xdev_standalone() {
```

Comment:
It looks like you copied this from the existing place - the existing place should be removed and just call this.